pkgs/tools/text/gawk/default.nix

```diff
@@ -46,6 +46,12 @@ stdenv.mkDerivation rec {
   configureFlags = [
     (if withSigsegv then "--with-libsigsegv-prefix=${libsigsegv}" else "--without-libsigsegv")
     (if interactive then "--with-readline=${readline.dev}" else "--without-readline")
+  ]
+  # nixpkgs on musl targets defaults to CFLAGS=-fPIE. That is incompatible with
+  # LDFLAGS=-no-pie that --enable-pma requests. Until gawk is fixed let's disable
+  # pma on affected targets.
```

Comment:
I don't think gawk can be "fixed", as PIE sounds like it would totally break gawk's ability to dump state out and load it back in.

Couldn't we just disable pie?

Reply:
https://github.com/NixOS/nixpkgs/pull/210136

Reply:
Yeah, I wanted to say "fixed" in terms or fully disabling PIE. I sent this thing upstream yesterday (but mailing lists took a while to catch up): https://lists.gnu.org/archive/html/bug-gawk/2023-01/msg00014.html